src/Tickets/Commerce/Admin/Orders_Page.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Handles displaying of orders page and sub menu.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Commerce\Admin
+ */
+
+namespace TEC\Tickets\Commerce\Admin;
+
+use TEC\Tickets\Commerce\Order;
+use Tribe\Admin\Pages;
+
+/**
+ * Class Orders_Page.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Commerce\Admin
+ */
+class Orders_Page {
+
+	/**
+	 * Event Tickets menu page slug.
+	 *
+	 * @var string
+	 */
+	public static $parent_slug = 'tec-tickets';
+
+	/**
+	 * Adds the Event Tickets Commerce Orders page.
+	 *
+	 * @since TBD
+	 */
+	public function add_orders_page() {
+		add_submenu_page(
+			static::$parent_slug,
+			esc_html__( 'Orders', 'event-tickets' ),
+			esc_html__( 'Orders', 'event-tickets' ),
+			Pages::get_capability(),
+			'edit.php?post_type=' . Order::POSTTYPE,
+			'',
+			1.7
+		);
+	}
+}
```

Comment:
This needs to be registered as a singleton, otherwise it will spawn a new instance every time, I would like to avoid the public static prop, would love to go all non-static methods and protected props to start.

A couple of other things:
1. Create a method for all 7 items used by `add_orders_page()`.
1. Lets make sure we have tests as we start here.
1. Handle the `edit.php` url using `add_query_arg` not string concat.
1. Make sure you save the return of the `add_submenu_page` to a prop on this class, so we can re-use, this particular class will need to have a bunch o bell an whistles before we are over here, since quite a bite o filtering will be needed.

Reply:
I'll update this. Thank you!